### PR TITLE
core/state: fix inconsistent verkle test error messages

### PR DESCRIPTION
**Pull request by hteevoli**
Some of the values for comparison and in error message are inconsistent.
Correct me if I chose the wrong one.

### CODE REVIEW

**Comment by hteevoli** [comment]
Hi, @gballet. Can you help review this PR? Thank you.

**Review by holiman — APPROVED** [review]
LGTM, a lot more readable and less prone to be incorrect

File: core/state/access_events_test.go [hunk 1]
```diff
@@ -71,8 +72,9 @@ func TestAccountHeaderGas(t *testing.T) {
 
 	// Check cold write cost
 	gas = ae.VersionGas(testAddr, true)
-	if gas != params.WitnessBranchWriteCost+params.WitnessChunkWriteCost {
-		t.Fatalf("incorrect gas computed, got %d, want %d", gas, params.WitnessBranchReadCost+params.WitnessBranchWriteCost)
+	want := params.WitnessBranchWriteCost+params.WitnessChunkWriteCost
+	if gas != want {
```
Comment (holiman) [inline core/state/access_events_test.go:76]:
```
core/state/access_events.go:1: : # github.com/ethereum/go-ethereum/core/state [github.com/ethereum/go-ethereum/core/state.test]
core/state/access_events_test.go:75:7: no new variables on left side of :=
core/state/access_events_test.go:88:7: no new variables on left side of := (typecheck)
// Copyright 2021 The go-ethereum Authors
util.go:47: exit status 1

```

File: core/state/access_events_test.go [hunk 2]
```diff
@@ -41,8 +41,9 @@ func TestAccountHeaderGas(t *testing.T) {
 
 	// Check cold read cost
 	gas := ae.VersionGas(testAddr, false)
-	if gas != params.WitnessBranchReadCost+params.WitnessChunkReadCost {
-		t.Fatalf("incorrect gas computed, got %d, want %d", gas, params.WitnessBranchReadCost+params.WitnessChunkReadCost)
+	want := params.WitnessBranchReadCost+params.WitnessChunkReadCost
+	if gas != want {
```
Comment (holiman) [inline core/state/access_events_test.go:45]:
```suggestion
	if want := params.WitnessBranchReadCost+params.WitnessChunkReadCost; gas != want {
```
And the same for all the cases further down

**Review by holiman — COMMENTED** [review]
Some nitpicks

**Review by MariusVanDerWijden — APPROVED** [review]
sgtm

**Comment by hteevoli** [comment]
Lints are weird. The original code pass the lint check?

**Review by gballet — APPROVED** [review]

**Pull request merged**